src/conditions/mod.rs

```diff
@@ -8,6 +9,8 @@ pub mod remap;
 
 pub use check_fields::CheckFieldsConfig;
 
+use self::remap::RemapConfig;
```

Comment:
Just a nit: I think the `self::` can be omitted here.

Reply:
Unfortunately, without that we get:

```
12 | use remap::RemapConfig;
   |     ^^^^^ ambiguous name
   |
   = note: `remap` could refer to a crate passed with `--extern`
   = help: use `::remap` to refer to this crate unambiguously
```